padl/dumptools/var2mod.py

```diff
@@ -556,18 +556,18 @@ def find_codenode(name: ScopedName):
 
 
 def _get_nodes_without_in_edges(graph):
-    """Get all nodes in directed graph *graph* that don't have incoming edges.
```

Comment:
In this entire file I think `graph` was accidentally refactored to `self`. Please change back.